ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -37,6 +38,10 @@ describeCommand
     : DESCRIBE tableSourceClause
     ;
 
+showCatalogsCommand
+    : SHOW CATALOGS
```

Comment:
Did you check if any this kind of command in SPL? Not sure if `SHOW CATALOG` is too SQL-ish.

Btw, any plan to support this in SQL?

Reply:
Yes, https://github.com/opensearch-project/sql/issues/946

Reply:
Yeah, Kusto Query Language has similar syntax. `.show TABLES`.  So I sticked to the same, also we recently introduced describe table recently.

Reply:
Got it. Could you check how SPL support this? Because I think SPL has been our main reference.